src/main/java/com/thealgorithms/datastructures/hashmap/hashing/MajorityElement.java

```diff
@@ -0,0 +1,39 @@
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+
+//import static org.junit.jupiter.api.Assertions.*;
+
+/*
+This class finds the majority element(s) in an array of integers.
+A majority element is an element that appears more than n/3 times, where n is the length of the array.
+*/
+public class MajorityElement {
+    /*
+    This method finds the majority element(s) in the given array of integers and prints them to console.
+    @param nums: an array of integers
+    */
+    public static void majority(int[] nums){
+        HashMap<Integer,Integer> map = new HashMap<>();
+        int n = nums.length;
+        for (int i = 0; i < n; i++) {
+            if (map.containsKey(nums[i])){ //true
+                map.put(nums[i],map.get(nums[i])+1);
+            }else { //false or element appeared first time
+                map.put(nums[i],1);
+            }
+        }
+        for (int key: map.keySet()) {
+            if (map.get(key) > n/3){
+                System.out.println(key);
```

Comment:
Please return it instead of printing to console

Reply:
done